Sources/Plasma/FeatureLib/pfConditional/plObjectInBoxConditionalObject.cpp

```diff
@@ -125,155 +124,160 @@ bool plObjectInBoxConditionalObject::Verify(plMessage* msg)
 // volume sensor conditional 
 //
 plVolumeSensorConditionalObject::plVolumeSensorConditionalObject() :
-fTrigNum(-1),
-fType(0),
-fFirst(false),
-fTriggered(false),
-fIgnoreExtraEnters(true)
+    fTrigNum(-1),
+    fType(0),
+    fFirst(false),
+    fTriggered(false),
+    fFlags(kIgnoreExtraEnters)
 {
     SetSatisfied(true);
 }
 
+void plVolumeSensorConditionalObject::IgnoreExtraEnters(bool ignore)
+{
+    if (ignore)
+        fFlags |= kIgnoreExtraEnters;
+    else
+        fFlags &= ~kIgnoreExtraEnters;
+}
+
+void plVolumeSensorConditionalObject::NoServerArbitration(bool arbitration)
```

Comment:
If the flag to set is `kNoServerArbitration`, the variable name should reflect that instead of the opposite.